tools/github-team-user-store/README.md

```diff
@@ -0,0 +1,28 @@
+# GitHub Team User Store
+
+## Overview
+
+The github-team-user-store is an internal only tool. The tool will get the list of teams/users under azure-sdk-write and store the results as a json string in Azure blob storage. The purpose of this is to enable team usage in CODEOWNERS by allowing the CodeOwnersParser to pull the list of users when it encounters a non-user, or team, entry. The reason why this code isn't directly in the CodeOwnersParser is because it requires a specific, fine-grained Personal Access Token (PAT) in order to be able to get the users for teams which isn't something that's readily available or prudent to be added to all of the places where CodeOwnersParser is used.
```

Comment:
So if I understand correctly running this tool puts the `teams <-> user logins` mapping into the blob store, which will be read upon runs of `CodeOwnersParser`. The mapping is read from the `azure-sdk-write` string.

My questions are: 
1. how are we supposed to populate `azure-sdk-write` string? 
2. how and how often are we supposed to execute this tool? Manually, by hand?
3. what will happens if the mapping is outdated because the tool wasn't yet run to reflect recent changes?

I think it would be cool if the README has sections answering all these questions.

Reply:
To answer the questions:
1. As previously mentioned, there is no azure-sdk-write string, it's the root team for that all azure sdk teams with write permission are supposed to be children of. The dictionary of team/users recursively includes all the teams under, and including, azure-sdk-write.
2. As stated in the PR description, this tool will run in a pipeline which has not been created yet. I plan on having it run at least once daily and we'll tweak accordingly.
3. Anything running that uses this data will run with the data it has at the time.